react/src/components/atoms/buttons/ButtonAlert/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import is from 'is';
+import PropTypes from 'prop-types';
+import './style.css';
+
+const ButtonAlert = ({
+  text, onClick, hideText, showText, classes
+}) => {
+  const buttonProps = {
+    className: buttonClasses
+  };
+
+  let buttonClasses = 'ma__button-alert';
+  if (classes && classes.length > 0) {
+    buttonClasses += ` ${classes}`;
+  }
+
+  if (is.fn(onClick)) {
+    buttonProps.onClick = onClick;
+  }
+  return(
+    <button {...buttonProps} className={buttonClasses} onClick={onClick}>
```

Comment:
onClick is defined within buttonProps and doesn't need to be set as a prop here. When buttonProps is destructured `{...buttonProps}` it gets `onClick` defined as an attribute onto the button tag. Same with className.
```suggestion
    <button {...buttonProps}>
```